freqtrade/persistence/models.py

```diff
@@ -1079,9 +1087,11 @@ class Trade(_DECL_BASE, LocalTrade):
     fee_open = Column(Float, nullable=False, default=0.0)
     fee_open_cost = Column(Float, nullable=True)
     fee_open_currency = Column(String(25), nullable=True)
+    fee_open_is_maker = Column(Boolean, default=True)
     fee_close = Column(Float, nullable=False, default=0.0)
     fee_close_cost = Column(Float, nullable=True)
     fee_close_currency = Column(String(25), nullable=True)
+    fee_close_is_maker = Column(Boolean, default=True)
```

Comment:
I'm not sure about the usefulness of these columns.

If you consider realistic order handling (the fee comes from the **filled order** - both in live and dry-run!) - then you'll not be able to reliably fill these columns (so the content may be wrong - and as such, the usage in the calculation may be wrong.